rust/src/parallel_map.rs

```diff
@@ -152,10 +152,10 @@ mod tests {
 
     #[test]
     fn plus_one() {
-        for iterator_length in 0 .. 50 {
-            for threads in 1 .. 20 {
+        for iterator_length in 0..50 {
```

Comment:
My bad for not running nightly. WDYT about having `rust-toolchain.toml` with:

```
[toolchain]
channel = "nightly"
```

Reply:
This is a project decision. Ideally, if you can use stable you should, because you will be able to reach more users.

You can still use nightly for development and both stable and nightly CI (using `cargo +nightly fmt -- --check` for formatting).

Reply:
Fantastic, thank you. I will do that and also create a `cargo fmt` workflow 0:-)

Reply:
Done in https://github.com/google/sedpack/pull/30